project/Dependencies.scala

```diff
@@ -43,13 +43,13 @@ object ExternalDependencies {
 
 object ServiceDependencies {
   import ExternalDependencies._
+  val common: Seq[ModuleID] = scalatest ++
+    logging ++
+    config ++
+    uPickle
```

Comment:
Are these not "dependencies of the project `common`

Reply:
Good point.  I've spent a bit more time than I should in the catalogue-pipeline wondering how a stage got a particular dependency.